<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="Numbers">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>181</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Move">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputNum">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>113</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>NUMBER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputIndex">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>113</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>INDEX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputFrom">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>FROM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTo">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>280</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TO</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
